beacon-chain/BUILD.bazel

```diff
@@ -40,7 +40,7 @@ go_image(
     goos = "linux",
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain",
     race = "off",
-    static = "off",  # Static enabled binary seems to cause issues with DNS lookup with cgo.
+    static = "on",  # Static enabled binary seems to cause issues with DNS lookup with cgo.
```

Comment:
these are the only changes?

Reply:
yes, turning it static seems to have resolved its docker issues